tests/cpp/test_circular_buffering.cpp

```diff
@@ -1237,9 +1243,8 @@ TEST_P(TmaCircularBufferingTest, Pointwise) {
   TensorView* tv3 = tv0->cacheAfter(LoadStoreOpType::CpAsyncBulkTensorTile);
   tv3->setMemoryType(MemoryType::Shared);
 
-  // Load TV1 into shared memory
   TensorView* tv4 = tv1->cacheAfter();
-  tv4->setMemoryType(MemoryType::Shared);
+  tv4->axis(0)->parallelize(ParallelType::BIDx);
```

Comment:
Just a minor refactor so it is easier to disable circular buffering with set op in my warp specialization PR.